class.photon.php

```diff
@@ -48,9 +48,11 @@ private function setup() {
 		if ( ! function_exists( 'jetpack_photon_url' ) )
 			return;
 
-		// Images in post content and galleries
+		// Images in post content, galleries, and bbPress forums.
 		add_filter( 'the_content', array( __CLASS__, 'filter_the_content' ), 999999 );
 		add_filter( 'get_post_galleries', array( __CLASS__, 'filter_the_galleries' ), 999999 );
+		add_filter( 'bbp_get_topic_content', array( __CLASS__, 'filter_the_content' ), 999999 );
```

Comment:
I would suggest these go in https://github.com/Automattic/jetpack/blob/master/3rd-party/bbpress.php